### PR TITLE
Change mastodon integration

### PR DESCRIPTION
**Pull request by egekorkan**
Trying with https://www.mastofeed.com/ to adjust the colours

### CODE REVIEW

**Comment by egekorkan** [comment]
Not sure why the width is so much. I will have a look later on.

File: docs/index.html [hunk 1]
```diff
@@ -87,9 +87,7 @@ <h3>Documentation</h3>
 	
 	</div>
 	<div class="col-md-4">
-		<iframe src="https://w3c.social/@wot/111613133105805601/embed" class="mastodon-embed" style="max-width: 100%; border: 0;"
-			width="400" allowfullscreen="allowfullscreen"></iframe>
-		<script src="https://w3c.social/embed.js" async="async"></script>
+		<iframe allowfullscreen sandbox="allow-top-navigation allow-scripts allow-popups allow-popups-to-escape-sandbox" width="400" height="800" src="https://www.mastofeed.com/apiv2/feed?userurl=https%3A%2F%2Fw3c.social%2Fusers%2Fwot&theme=light&size=100&header=false&replies=false&boosts=false"></iframe>
```
[the remark this answers is not in the document]
Comment (egekorkan) [reply docs/index.html:90]: Yup we used to have 400 as well. I can just adjust to match the look of the previous case.

Comment (danielpeintner) [inline docs/index.html:90]:
```suggestion
		<iframe allowfullscreen sandbox="allow-top-navigation allow-scripts allow-popups allow-popups-to-escape-sandbox" style="min-width: 180px; width: 100%" height="800" src="https://www.mastofeed.com/apiv2/feed?userurl=https%3A%2F%2Fw3c.social%2Fusers%2Fwot&theme=light&size=100&header=false&replies=false&boosts=false"></iframe>
```

Comment (danielpeintner) [reply docs/index.html:90]: Note: The height I think should be smaller also

**Comment by egekorkan** [comment]
I have tested locally and it looks good. Also reduced height. There is still a frame around it but to fix it, we need to host all this ourselves and copy the code.

**Comment by egekorkan** [comment]
I have tried to remove the frame but I think my CSS knowledge hit its limits. We can merge this and open an issue about that issue. It is still better than the current black box.

**Review by danielpeintner — APPROVED** [review]

**Comment by lu-zero** [comment]
```css
.mastodon-embedding {
  border: none;
}
```

image


might be enough, hiding the scrollbar on the other hand is a bit annoying to support across browsers.

**Comment by egekorkan** [comment]
Call of 23.01: We have added the proposal of Luca. We can iterate on this by using https://sampsyo.github.io/emfed/ to have more control on the styling.

**Pull request merged**